share/pmacc/examples/gameOfLife2D/main.cpp

```diff
@@ -41,7 +41,7 @@ int main( int argc, char **argv )
 {
     typedef ::gol::Space Space;
 
-    std::vector<uint32_t> devices;  /* will be set by boost program argument option "-d 3 3 3" */
+    std::vector<uint32_t> devices;  /* will be set by boost program argument option "-d 3 3" */
```

Comment:
This is unrelated, but was noticed during this change. The example only supports 1d and 2d